SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -23,13 +23,16 @@ enum ESchemaComponentType : int32
 	SCHEMA_OwnerOnly,
 	SCHEMA_Handover,
 
+	SCHEMA_Count,
+
 	// RPCs
 	SCHEMA_ClientRPC,
```

Comment:
Would you want to change this to be `SCHEMA_ClientReliableRPC` (Ditto for server)

Reply:
Done